Tests/Functional/TestCaseLifecycle/main.swift

```diff
@@ -21,6 +21,12 @@ class SetUpTearDownTestCase: XCTestCase {
 
     var value = 0
 
+// CHECK: In class setUp\(\)
```

Comment:
With #94 merged you may now indent the `// CHECK:`, like so:

```swift
func myFunc() {
    // CHECK: foo
    print("foo")
    if (bar) {
        // CHECK: baz
        print("baz")
    }
}
```

I think indenting is more readable, so I'd encourage you to do that! (I had been meaning to start sending out pull requests to indent the checks in the tests, but forgot 😇 )